packages/plugin-commands-env/src/constants.ts

```diff
@@ -0,0 +1 @@
+export const DEFAULT_NODE_MIRROR = 'https://nodejs.org/download'
```

Comment:
I don't see many `constants` files in the repo. Is this something we want to discourage? I moved this here to make it more consistently shared between `resolveNodeVersion` and `installNode`.

Reply:
it is ok